src/main/java/com/notably/logic/parser/suggestion/OpenSuggestionCommandParser.java

```diff
@@ -23,7 +23,7 @@
 
     private static final String RESPONSE_MESSAGE = "Open a note";
     private static final String RESPONSE_MESSAGE_WITH_TITLE = "Open a note titled \"%s\"";
-    private static final String RESPONSE_MESSAGE_CANNOT_OPEN_NOTE = "Open delete \"%s\". Invalid path.";
+    private static final String RESPONSE_MESSAGE_CANNOT_OPEN_NOTE = "\"%s\" is an invalid path";
```

Comment:
Should we phrase this as `"Cannot open \"%s\" as it is an invalid path"`?

Also note that your suggestion text for the delete command is inconsistent with this!

Reply:
Ok!